socs/agents/wiregrid_tiltsensor/agent.py

```diff
@@ -99,8 +112,25 @@ def acq(self, session, params=None):
                 # data taking
                 current_time = time.time()
                 msg, angles = self.tiltsensor.get_angle()
-                if self.sensor_type == 'sherborne':
-                    msg, temperatures = self.tiltsensor.get_temp()
```

Comment:
This change drops the definition of `temperatures`, which is later accessed down on line 139+140. It's now undefined which is why [pre-commit is failing](https://results.pre-commit.ci/run/github/186511668/1757500927.VV05wgo-SXm3lHCPwuGEAQ). This needs to be addressed before merging.